### ISSUE TITLE
werf bundle render does not use secret-values

### ISSUE THREAD

**Issue by CRASH-Tech** [werf/werf#4600]
### Before proceeding

- [X] I didn't find a similar [issue](https://github.com/werf/werf/issues)

### Version

v1.2.122

### How to reproduce

werf bundle render

### Result

Error: helm templates rendering failed: template: test/templates/005-worker.yaml:16:19: executing "test/templates/005-worker.yaml" at <$.Values.mysql.password>: nil pointer evaluating interface {}.password

### Expected result

werf print manifests with secrets

### Additional information

<img width="1124" alt="image" src="https://user-images.githubusercontent.com/13507148/178453475-7df2712c-2a83-411f-9736-480b3d2b4343.png">

**Comment by distorhead** [comment]
werf intentionally does not store content of `.helm/secret-values.yaml` in the bundle in container registry.

Though there is a BUG: bundle render command does not have --secret-values param to supply secret values during rendering of an already published bundle the same way as bundle apply:

```
export WERF_SECRET_KEY=...
werf bundle apply  --repo REPO --secret-values ./my-secret-values.yaml
werf bundle render --repo REPO --secret-values ./my-secret-values.yaml
```

**Important!** When running bundle render from inside git-repository it should work the same way as bundle publish command, which does not use default `.helm/secret-values.yaml` file. In such case you could use explicit param `werf bundle render --secret-values .helm/secret-values.yaml` if you need it.

We will add secret-values param for bundle render command ASAP.

**Comment by distorhead** [comment]
@CRASH-Tech Hi! `--secret-values` option for the `werf bundle render` available since `v1.2.126` (alpha and beta channels at the moment).
